podcasts/DownloadManager.swift

```diff
@@ -231,6 +231,45 @@ class DownloadManager: NSObject, FilePathProtocol {
         }
     }
 
+    func downloadParallelToStream(of episode: BaseEpisode) -> AVPlayerItem? {
+        guard let playbackItem = PlaybackItem(episode: episode).createPlayerItem() else {
+            return nil
+        }
+
+        guard FeatureFlag.cachePlayingEpisode.enabled,
+              !episode.videoPodcast(),
+              let urlAsset = playbackItem.asset as? AVURLAsset,
+              !urlAsset.url.isFileURL // only  start download if it's a remote file that we are playing
+        else {
+            return playbackItem
+        }
+        #if !os(watchOS)
+        Task {
+            //TODO: check for the following: download is not happening already because of auto download, cancel the auto download and use this file
+            episode.autoDownloadStatus = AutoDownloadStatus.playerDownloadedForStreaming.rawValue
+            episode.contentType = UTType.mpeg4Audio.preferredMIMEType
+            DataManager.sharedManager.save(episode: episode)
+
+            NotificationCenter.postOnMainThread(notification: Constants.Notifications.episodeDownloadStatusChanged, object: episode.uuid)
+
+            let outputURL = URL(fileURLWithPath: streamingBufferPathForEpisode(episode), isDirectory: false)
+            //TODO: force contentType to M4A so downloads paths match on the remaing code
+
+            downloadingEpisodesCache[episode.uuid] = episode
+            FileLog.shared.addMessage("Media Export Session -> starting exporting: \(episode.title ?? "")")
```

Comment:
[nit]

I think we can use the standard we use in other log messages, ie.:

```
DownloadManager export session: start exporting <episode uuid>
```

(episode UUID is easier for us to locate)